admin-center/display/govbox-jobs.md

```diff
@@ -21,4 +28,4 @@ Searchers will also see a direct link to [USAJobs.gov](https://www.usajobs.gov)
 
 ---
 
- **Did you know?** Are you a federal agency that doesn't post your jobs on USAJobs? We want your jobs data. Email us at <search@support.digitalgov.gov> to let us know where you publish your job openings so that we can include them in the Job Openings module.
+If you would like to show [job openings]({{ site.baseurl }}/manual/govbox-jobs.html) in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```

Comment:
just realized that the link in this line points back to this very page. Please remove, thanks!